### DISCUSSION TITLE
Changelog littered with "pre-release" notes

### DISCUSSION THREAD

**Discussion by virtuallyvlad** [standardnotes/forum#2855] [Suggestions] [answered]
Is there a way to hide all of the "pre-release" notes that now litter the [changelog](https://github.com/standardnotes/app/releases)?

And/or is there a more clean version of the changelog?  As of late it's become too cumbersome to understand what's going on with the updates; for example, you'd have to read through 6 pages just to know what was updated from July 7 to August 13.

**Comment by virtuallyvlad** [comment]
Ok so searching `prerelease:false` will hide those.  But I still think it'd be better to have pre-release notes hidden/elsewhere by default, so that the release notes are more readable for most users (who won't know that search parameter).  Unless there's another place I'm not aware of where release notes are abbreviated/summarized.

**Comment by myreli** [comment, marked as answer]
Release notes are now summarized in the new "What's New" section of the app:

<img width="1467" alt="Screenshot 2023-02-25 at 12 17 20" src="https://user-images.githubusercontent.com/17554234/221364904-ef139799-8508-4d0f-9bf3-eb4220ff704f.png">

**Comment by virtuallyvlad** [comment]
Yea I think that fulfills this request, thank you.  Marked as answer.
